src/database/schema/ChatSchema.ts

```diff
@@ -288,19 +327,27 @@ class ChatSchema implements BaseDbSchema {
       user: null,
       userId,
       isMe: true,
-      isContinuous: true
+      isContinuous: true,
+      attachmentJson: JSON.stringify(attachments)
     });
   }
 
   static fromInitAnonymousChatAPI(data: ModifyAnonymousChatData, status = 'sent'): ChatSchema {
     let rawJson = null;
+    let attachmentJson = null;
 
     try {
       rawJson = JSON.stringify(data);
     } catch (e) {
       console.log('error stringify');
       console.log(e);
     }
+    try {
+      attachmentJson = JSON.stringify(data?.message?.attachments);
+    } catch (e) {
+      console.log('error stringify');
+      console.log(e);
+    }
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Similar to previous comments, add more context to the error logs.



<!-- This is an auto-generated comment by OSS CodeRabbit -->